src/components/popover/popover.js

```diff
@@ -32,7 +32,7 @@ export default {
     },
     placement: {
       type: String,
-      default: 'right'
+      default: 'auto'
```

Comment:
Bootstrap V4 uses a default of 'right' for their popovers.

Reply:
Although I was never overly fond of it being 'right'

Reply:
It's fine to match Bootstrap V4, I fussed with it after noticing the docs didn't match the actual as-defined default. I'll change it to match V4.